source/renderer/app/api/api.js

```diff
@@ -203,38 +212,87 @@ export default class AdaApi {
     this.config = config;
   }
 
-  getWallets = async (): Promise<Array<Wallet>> => {
+  /* eslint-disable */
+  getWallets = async () => {
```

Comment:
@daniloprates api response is prevented and wallets are faked. Can you remove testing / faked data